numpy/core/_add_newdocs.py

```diff
@@ -5243,8 +5243,6 @@
 
     Examples
     --------
-    >>> np.core.umath_tests.matrix_multiply.signature
-    '(m,n),(n,p)->(m,p)'
     >>> np.linalg._umath_linalg.det.signature
     '(m,m)->()'
     >>> np.add.signature is None
```

Comment:
```suggestion
    >>> np.matmul.signature
    '(n?,k),(k,m?)->(n?,m?)'
    >>> np.add.signature is None
```
A bit more complicated with the ?, but maybe that is fine...